pycheribuild/config/chericonfig.py

```diff
@@ -289,6 +289,9 @@ def __init__(self, loader: ConfigLoaderBase, action_class):
             help="Perform a shallow `git clone` when cloning new projects. This can save a lot of time for large"
             "repositories such as FreeBSD or LLVM. Use `git fetch --unshallow` to convert to a non-shallow clone")
 
+        self.fpga_custom_env_setup_script = loader.add_path_option("fpga-env-setup-script",
```

Comment:
```suggestion
        self.fpga_custom_env_setup_script = loader.add_path_option("cheri-mips-fpga-env-setup-script",
```
In case we need something similar for riscv